test_unstructured/partition/test_html_partition.py

```diff
@@ -723,3 +726,43 @@ def test_partition_html_with_table_without_tbody(tag: str, expected: str):
     )
     partitions = partition_html(text=table_html)
     assert partitions[0].metadata.text_as_html == expected
+
+
+@pytest.fixture
+def partitioned_html_with_duplicate_elements():
+    filename = "example-docs/fake-html-duplicates.html"
+    with open(filename) as f:
+        elements = partition_html(file=f)
+    return elements
```

Comment:
This resolves to:
```python
return partition_html("example-docs/fake-html-duplicates.html")
```
which can be inlined in the tests that use it so folks don't have to go looking for the fixture..

Reply:
done, can be resolved